src/gui/torrentcreatordlg.ui

```diff
@@ -273,6 +273,16 @@
         </property>
        </widget>
       </item>
+      <item row="3" column="0">
+       <widget class="QLabel" name="labelSource">
+        <property name="text">
+         <string>Source:</string>
```

Comment:
What "Source" does mean here?

Reply:
https://github.com/qbittorrent/qBittorrent/issues/7965#issuecomment-349292327

Reply:
>#7965 (comment)

Basically that, its a custom string field under "info" dict named "source" commonly used by private seeders/trackers.